src/test/java/moduletwo5/page/constants/StringСonstants.java

```diff
@@ -0,0 +1,22 @@
+package moduletwo5.page.constants;
+
+public class StringСonstants {
```

Comment:
This is a good idea to store constants in a separate Class. But I'd also suggest restricting the creation of this class objects (e.g. private constructor)